packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -181,37 +70,189 @@ export const SignOnPage = () => {
         <IconCloseAlt color='staticWhite' />
       </Link>
       <BackgroundWaves zIndex={0} />
-      <Paper w='100%'>
-        <Flex
+      <Paper
+        w='100%'
+        css={{
+          flex: 1,
+          maxWidth: 1440,
+          minWidth: 744,
+          overflow: 'hidden'
+        }}
+      >
+        <ScrollView
           direction='column'
-          w={isExpanded ? '100%' : 480}
           h='100%'
+          w={isExpanded ? '100%' : 584}
+          flex={isExpanded ? undefined : '1 0 0'}
           css={{
-            position: 'absolute',
-            top: 0,
-            left: 0,
+            maxWidth: isExpanded ? '100%' : 584,
+            minWidth: 400,
             background: 'white',
             zIndex: 1,
             transition: `width ${motion.expressive}`
           }}
         >
-          {routes}
-        </Flex>
+          {children}
+        </ScrollView>
         <Flex
           alignItems='center'
           justifyContent='center'
+          flex='1 0 0'
           css={{
-            marginLeft: 480,
-            flexGrow: 1,
+            // flexGrow: 1,
```

Comment:
leaving this commented out for any particular reason?

Reply:
no good find, thanks